node/core/core.go

```diff
@@ -0,0 +1,38 @@
+package core
+
+import (
+	"go.uber.org/fx"
+
+	"github.com/celestiaorg/celestia-node/core"
+)
+
+// Config combines all configuration fields for Core subsystem.
+type Config struct {
+	EnableRemote bool
```

Comment:
Can we just call this `Remote` and have it be a bool.

Reply:
Agree, done